src/test/java/org/opensearch/neuralsearch/query/NeuralSparseQueryIT.java

```diff
@@ -114,6 +141,47 @@ public void testBasicQueryUsingQueryTokens() {
         }
     }
 
+    /**
+     * Tests basic query with boost:
+     * {
+     *     "query": {
+     *         "neural_sparse": {
+     *             "text_sparse": {
+     *                 "query_tokens": {
+     *                     "hello": float,
+     *                     "world": float,
+     *                     "a": float,
+     *                     "b": float,
+     *                     "c": float
+     *                 },
+     *                 "boost": 2
+     *             }
+     *         }
+     *     }
+     * }
+     */
+    @SneakyThrows
+    public void testBasicQueryUsingQueryTokens_whenTwoPhaseEnabled() {
```

Comment:
In these test cases, we always get same scores no matter 2-phase is enabled or disabled. And this can't help us find out when there is problem in 2-phase.
Can we design some test cases where we have different scores for 2-phase enabled/disabled? We can just write raw token scores and use raw token scores to search.

Reply:
After we have different behavior for 2-phase enabled/disabled, we should add test cases about update cluster settings to control the behavior, and use DSL to control the behavior (and DSL should overwrite cluster settings)

Reply:
Add a test case for this: testNeuralSparseQuery_whenDifferentTwoPhaseParameter_thenGetDifferentResult.
Use different parameter to get expected different result.